powertools-large-messages/src/main/java/software/amazon/lambda/powertools/largemessages/internal/LargeMessageAspect.java

```diff
@@ -0,0 +1,48 @@
+package software.amazon.lambda.powertools.largemessages.internal;
+
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Pointcut;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.lambda.powertools.largemessages.LargeMessage;
+
+import java.util.Optional;
+
+/**
+ * Handle {@link LargeMessage} annotations.
+ */
+@Aspect
+public class LargeMessageAspect {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LargeMessageAspect.class);
+
+    @SuppressWarnings({"EmptyMethod"})
+    @Pointcut("@annotation(largeMessage)")
+    public void callAt(LargeMessage largeMessage) {
```

Comment:
Do we need this at all if we're not using it - is it needed because of the reference on :26?

Reply:
yes, we need it, this is how it works...